spec/lib/utils/job-utils/racadm-parser-spec.js

```diff
@@ -0,0 +1,151 @@
+            expect(result.BIOS).to.deep.equal({
+                FQDD: 'BIOS.Setup.1-1',
+                installationDate: '2015-11-26T06:54:10Z',
+                currentVersion: '1.0.3',
+                rollbackVersion: '1.0.3',
+                availableVersion: '1.0.3',
+                elementName: 'BIOS'
+            });
+            expect(result.Disk1).to.deep.equal({
+                FQDD: 'Disk.Bay.1:Enclosure.Internal.0-0:RAID.Slot.1-1',
+                installationDate: '2015-11-26T07:28:09Z',
+                currentVersion: 'TS04',
+                elementName: 'Disk 1 in Backplane 0 of RAID Controller in Slot 1',
+                availableVersion: '',
+                rollbackVersion: ''
+            });
+            expect(result.Disk2).to.deep.equal({
+                FQDD: 'Disk.Bay.2:Enclosure.Internal.0-0:RAID.Slot.1-1',
+                installationDate: '2015-11-26T07:28:09Z',
+                currentVersion: 'TS04',
+                elementName: 'Disk 2 in Backplane 0 of RAID Controller in Slot 1',
+                availableVersion: '',
+                rollbackVersion: ''
+            });
+        });
+    });
+
+/*
+    describe("XML to Json Parser", function() {
+        it("should convert XML file to Json format", function() {
+            var data = racadmOutMock.racadmXml;
+            console.log(typeof data);
+            var result = parser.xmlToJson(data);
+
+        });
+    });
+*/
+
+    describe("Parse file path", function() {
+        it("should parse  remote filename and path", function() {
+            var data = "//192.168.191.207/share/bios.xml";
+            var result = parser.getPathFilename(data);
+            expect(result.path).to.equal('//192.168.191.207/share');
+            expect(result.name).to.equal('bios.xml');
+            expect(result.style).to.equal('remote');
+        });
+
+        it("should parse  remote filename and path", function() {
+            var data = "/home/share/bios.xml";
+            var result = parser.getPathFilename(data);
+            expect(result.path).to.equal('/home/share');
+            expect(result.name).to.equal('bios.xml');
+            expect(result.style).to.equal('local');
+        });
+
+        it("should report path format incorrect", function() {
+            var data = "home/share/bios.xml";
+            expect(function() {
+                parser.getPathFilename(data);
+            }).to.throw(Error);
+        });
+
+    });
+
+    describe("Parse  JID", function() {
+        it("should filter JID from console standard output", function() {
+            var data = racadmOutMock.racadmJobIdData;
+            var result = parser.getJobId(data);
+            expect(result).to.equal('JID_541335487816');
+        });
+
+        it("should report can't parser JID", function() {
+            var data = 'any string';
+            expect(function() {
+                parser.getPathFilename(data);
+            }).to.throw(Error);
+        });
```

Comment:
Could ghe Error report can't paser JID or further comparison of the Error is needed?